src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -44,6 +49,24 @@ class ClaimSimulation extends Simulation {
     randomChar.toString
   }
 
+  private def addressStep =
+    if (features.getOrElse("ADDRESS_LOOKUP", false)) {
```

Comment:
Weren't we using the toggle format of "ADDRESS_LOOKUP_ENABLED"?

Reply:
Fixed.